pyOCD/target/target_k20d50m.py

```diff
@@ -30,4 +31,5 @@ class K20D50M(Kinetis):
     def __init__(self, link):
         super(K20D50M, self).__init__(link, self.memoryMap)
         self.mdm_idr = 0x001c0000
+        self._svd_location = SVDFile(vendor="Freescale", filename="MK20D5.xml", is_local=False)
```

Comment:
A pretty wide selection of SVD files ship with the `cmsis_svd` package.  New SVDs can be added by sending PRs to the cmsis-svd project: https://github.com/posborne/cmsis-svd/tree/master/data